### PR TITLE
Add `alias` to PatchTeaser

### PR DESCRIPTION
**Pull request by xphoniex**
Screenshot from 2023-05-12 09-29-24

### CODE REVIEW

File: src/components/Authorship.svelte [hunk 1]
```diff
@@ -3,6 +3,7 @@
   import { formatNodeId, formatTimestamp } from "@app/lib/utils";
 
   export let authorId: string;
+  export let authorAlias: string;
```
Comment (rudolfs) [inline src/components/Authorship.svelte:6]:
```suggestion
  export let authorAlias: string | undefined = undefined;
```

File: src/components/Authorship.svelte [hunk 2]
```diff
@@ -35,9 +36,15 @@
...
   <span class="id layout-desktop">
     {formatNodeId(authorId)}
+    {@html authorAlias
+      ? '<span class="alias">(' + authorAlias + ")</span>"
+      : ""}
   </span>
   <span class="id layout-mobile">
     {formatNodeId(authorId).replace("did:key:", "")}
+    {@html authorAlias
+      ? '<span class="alias">(' + authorAlias + ")</span>"
```
Comment (xphoniex) [inline src/components/Authorship.svelte:46]: when I add a styling for `"alias"` class it doesn't render, e.g.:
```
<style>
...
.alias {
  color: red !important;
}
</style>
```

any idea why?

File: src/components/Authorship.svelte [hunk 3]
```diff
@@ -35,9 +36,15 @@
...
+      : ""}
```
Comment (rudolfs) [inline src/components/Authorship.svelte:47]:
```suggestion
    {#if authorAlias}
    <span class="alias">{authorAlias}</span>
    {/if}
```

File: src/components/Authorship.svelte (hunk 2 above)
Comment (rudolfs) [reply src/components/Authorship.svelte:46]: Not sure, but could be because you're rendering with `@html`.

Comment (xphoniex) [reply src/components/Authorship.svelte]: yes, works now without `@html`.

File: src/components/Authorship.svelte [hunk 4]
```diff
@@ -27,6 +28,9 @@
   .body {
     white-space: nowrap;
   }
+  .alias {
+	  color: gray;
```
Comment (rudolfs) [inline src/components/Authorship.svelte:32]:
```suggestion
  var(--color-foreground-6)
```

Comment (xphoniex) [reply src/components/Authorship.svelte]: that's not the same color

Comment (xphoniex) [reply src/components/Authorship.svelte]: `5` works

Comment (xphoniex) [reply src/components/Authorship.svelte]: https://github.com/radicle-dev/radicle-interface/compare/647bf9c9d5eee91f89bd059d70aed86eb042f196..78a0d79c49f9110fb40997f0fbbca7040ddb5780

Comment (rudolfs) [reply src/components/Authorship.svelte]: Then pick one that matches from our color system: https://github.com/radicle-dev/radicle-interface/blob/master/public/colors.css

Comment (rudolfs) [reply src/components/Authorship.svelte]: You can see all available colors in dev mode by pressing <kbd>d</kbd> in the app.

Screenshot 2023-05-12 at 08 17 40

**Comment by rudolfs** [comment]
How do you add an alias via the CLI?

**Comment by xphoniex** [comment]
> How do you add an alias via the CLI?

```
$ rad track z6MksFqXN3Yhqk8pTJdUGLwATkRfQvwZXPqR2qMEhbS9wzpT --alias cloudhead
```

**Comment by rudolfs** [comment]
I'm running `radicle-httpd` version 0.1.0-7037b52 locally. Cloned heartwood with `rad clone` and added a couple of aliases, but they don't show up in the patch list. What am I doing wrong?

**Comment by xphoniex** [comment]
> I'm running `radicle-httpd` version 0.1.0-7037b52 locally. Cloned heartwood with `rad clone` and added a couple of aliases, but they don't show up in the patch list. What am I doing wrong?

you need to compile off this patch
```
7e9cb11  httpd: Add `alias` field to `/patches` and `/issues` routes
```

**Pull request merged**